octobot_trading/personal_data/orders/order_util.py

```diff
@@ -144,6 +144,49 @@ async def get_pre_order_data(exchange_manager, symbol: str, timeout: int = None,
     return currency_available, market_available, market_quantity, mark_price, symbol_market
 
 
+def get_futures_max_order_size(exchange_manager, symbol, side, current_price, reduce_only,
```

Comment:
We don't check in market status? Limits are very strict, it is quite easy to reach them.

Reply:
We need to check it after yes, this returns the theoretical max size that could be used in order

Reply:
ok 👍